### DISCUSSION TITLE
Disable navigation with arrow keys on some pages

### DISCUSSION THREAD

**Discussion by Shotgunosine** [executablebooks/meta#761] [Q&A] [answered]
We've got a jupyter book with a JSON validator on one page: https://bids-standard.github.io/stats-models/validator.html

It's currently quite hard to use because the left and right arrow keys navigate to other pages, keeping them from being used while editing text. Is there any way to disable arrow key navigation for an individual page?

**Comment by rowanc1** [comment, marked as answer]
I have had to do this here by patching into the static path:
https://github.com/executablebooks/mystjs/blob/main/docs/Makefile#L3

There is probably a better way to do it but I don't know how. At least will give you some places to start looking in:
`/_build/html/_static/documentation_options.js`

**Reply by Shotgunosine** [reply]
That works, thanks!
